harness/determined/tensorboard/base.py

```diff
@@ -174,3 +174,4 @@ def close(self) -> None:
         while self.is_alive():
             logging.info("Waiting for Tensorboard files to finish uploading")
             self.join(10)
+        logging.info("Tensorboard upload completed")
```

Comment:
I approve this pr.

But, while you are touching it, can you shuffle this code around so it doesn't basically always wait 10 seconds?  The way it works now there's a race condition that it almost always loses, even if there's no upload happening.

Reply:
disregard, this already has been fixed, I wasn't remembering correctly or thinking critically.